x/epochs/README.md

```diff
@@ -131,26 +129,20 @@ type Keeper interface {
 The `BeforeEpochStart` hook does different things depending on the identifier.
 
 If in a `day` identifier it:
+
 1. begins unbondings
 2. sweeps unbonded tokens to the redemption account
 3. cleans up old records
 4. creates empty epoch unbonding records for the next day
 
-If in a `stride_epoch` identifier it:
-5. creates and deposits records on each host zone
-6. sets withdrawal addresses
-7. updates redemption rates (if the epoch coincides with the correct interval)
-8. processes `TRANSFER` deposit records to the delegation Interchain Account (if the epoch coincides with the correct interval)
-9. processes `STAKE` deposit records to the delegation Interchain Account (if the epoch coincides with the correct interval)
-10. Query the rewards account using interchain queries, with the transfer callback to a delegation account as a staked record (if at proper interval)
+If in a `stride_epoch` identifier it: 5. creates and deposits records on each host zone 6. sets withdrawal addresses 7. updates redemption rates (if the epoch coincides with the correct interval) 8. processes `TRANSFER_QUEUE` deposit records to the delegation Interchain Account (if the epoch coincides with the correct interval) 9. processes `DELEGATION_QUEUE` deposit records to the delegation Interchain Account (if the epoch coincides with the correct interval) 10. Query the rewards account using interchain queries, with the transfer callback to a delegation account as a staked record (if at proper interval)
```

Comment:
I know it doesn't affect the actual rendering but it's a little easier to consume the markdown if the newlines are still there between each number

Reply:
hm odd I think this was a linter thing